plugins/user-settings-backend/src/deprecated.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import express from 'express';
+import { DatabaseService } from '@backstage/backend-plugin-api';
+import { IdentityApi } from '@backstage/plugin-auth-node';
+import { SignalsService } from '@backstage/plugin-signals-node';
+
+import { createRouter as internalCreateRouter } from './service';
+
+/**
+ * Type for the options passed to the "createRouter" function.
+ *
+ * @public
+ * @deprecated This type is only exported for legacy reasons and will be removed in the future.
+ */
+export type RouterOptions = {
+  database: DatabaseService;
+  identity: IdentityApi;
```

Comment:
i know this is mostly a formality (they are identical), but could use `IdentityService` from plugin-api for uniformity here

Reply:
Done 🙌🏻